MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -1635,6 +1638,19 @@ protected void writeToXmlEnd(PrintWriter pw1, int indent) {
 
             pw1.println(MekHqXmlUtil.indentStr(indent+1) + "</transportLinkages>");
         }
+        
+        if (numPlayerMinefields.size() > 0) {
+            MekHqXmlUtil.writeSimpleXMLOpenIndentedLine(pw1, indent + 1, "numPlayerMinefields");
+
+            for (int key : numPlayerMinefields.keySet()) {
+                MekHqXmlUtil.writeSimpleXMLOpenIndentedLine(pw1, indent + 2, "numPlayerMinefield");
+                MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 3, "minefieldType", key);
+                MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 3, "minefieldCount", numPlayerMinefields.get(key).toString());
+                MekHqXmlUtil.writeSimpleXMLCloseIndentedLine(pw1, indent + 2, "numPlayerMinefield");
+            }
+
+            MekHqXmlUtil.writeSimpleXMLCloseIndentedLine(pw1, indent + 1, "numPlayerMinefields");
```

Comment:
Could swap to indent++ and --indent on the open/close tags for ease of reading (in my eye, at least)

Reply:
All the other ones in the file look the same as these, so let's leave them until the whole file can be converted.